web-services/map-reduce/src/main/java/datawave/webservice/mr/configuration/MapReduceJobConfiguration.java

```diff
@@ -284,7 +288,17 @@ protected void prepareClasspath(String jobId, Job job, Path jobDir) throws Excep
         File[] jarFiles = libDir.listFiles(jarFilter);
         if (jarFiles != null) {
             for (File jar : jarFiles) {
-                addSingleFile(jar, new Path(classpath, jar.getName()), jobId, job, fs);
+                // remove guava classes from jboss-client.jar
+                if (jar.getName().equals("jboss-client.jar")) {
+                    List<Pattern> patterns = new ArrayList<>();
```

Comment:
ugh, but ok for now I guess....

Reply:
I know, but the alternative is storing a modified version of that file in our version control.  That's not great either.  The dependency is needed for our MapReduce jobs to run, but because of the inability to order the user classpath to the job we have to eliminate the newer guava classes that are shaded into that jar.  

I considered adding configuration to take care of this (which jar to transfer, which regexes to filter out).  I can do that later.  The java would be cleaner at least.   I ran out of time at the end of the week and this will work for now.